<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>440</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>330</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>小ai签票</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/img/logo.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="content_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>310</width>
      <height>380</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QWidget" name="title" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>310</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>SizeAllCursor</cursorShape>
     </property>
     <widget class="Mylabel" name="icon_close">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>5</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="icon_min">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>3</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginBtn" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>310</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <widget class="QLabel" name="loginBtn_text">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main_content" native="true">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>30</y>
       <width>305</width>
       <height>300</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="userName">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>100</y>
        <width>260</width>
        <height>30</height>
       </rect>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="markdown">
       <string/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="userPsw">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>150</y>
        <width>260</width>
        <height>30</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QLabel" name="textLogin">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>72</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Mylabel</class>
   <extends>QLabel</extends>
   <header location="global">components/close_label_btn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
